resources/views/scheduleInstrument.html

```diff
@@ -0,0 +1,492 @@
+                }
+                instrumentDropDown += '</select>';
+                jQuery('#instrumentDropDown').html(instrumentDropDown);
+                loadInstrumentSchedule(function(instrumentSchedule) {
+                    let currentYear = new Date().getFullYear();
+                    calendar = new Calendar('#calendar', {
+                        startDate: new Date(),
+                        numberMonthsDisplayed: 1,
+                        enableRangeSelection: LABKEY.user.canInsert,
+                        dataSource: instrumentSchedule,
+                        style: 'custom',
+                        selectRange: function(e) {
+                            editEvent({
+                                startDate: e.events.length ? e.events[0].startDate : e.startDate,
+                                endDate: e.events.length ? e.events[0].endDate : e.endDate,
+                                id:  e.events.length ? e.events[0].id : null,
+                                name:  e.events.length ? e.events[0].name : null,
+                                notes:  e.events.length ? e.events[0].notes : null
+                            });
+                        },
+
+                    });
+                    calendar.setCustomDayRenderer(function (element, date) {
+                        let events = calendar.getEvents(date);
+                        if (events && events.length) {
+                            let e = events[0];
+                            // Any time scheduled for the current project shows in a non-gray color
+                            let style =
+                                    'display: block; ' +
+                                    'width: 100%; ' +
+                                    'height: 40%; ' +
+                                    'background-color: ' + (e.project == project ? e.color : 'gray') + '; ' ;
+                            let div = document.createElement('div');
+                            div.style = style;
+                            let dateStr = e.startDate.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}) + ' - ' + e.endDate.toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'})
+                            div.innerHTML = dateStr + '&nbsp' + e.project;
+                            element.appendChild(div);
+
+                            element.addEventListener('click', function() {
+                                editEvent(e);
+                            });
+
+                            // add mouse on element
+
+                            element.addEventListener('mouseover', function() {
+                                var content = '';
+
+                                for(var i in events) {
+                                    // format 12/01/24, 10:00 AM - 12/01/24, 11:00 AM
```

Comment:
Let's respect the server's configured date/time formats instead of hard-coding here. See LABKEY.container.formats

Reply:
changed